FirebaseSegmentation/Sources/FIRSegmentation.m

```diff
@@ -12,4 +12,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#import "FIRSegmentation.h"
+#import "FirebaseSegmentation/Sources/Public/FIRSegmentation.h"
+
+#import "FIRAppInternal.h"
```

Comment:
Should be #import <FirebaseCore/FIRAppInternal.h> for this and other Core Private headers.

Reply:
Done.